ci-operator/step-registry/cli-manager-extension/cli-manager-extension-commands.sh

```
@@ -0,0 +1,123 @@
+if oc get namespace ${OPERATOR_NAMESPACE} &>/dev/null; then
+  echo "Operator namespace ${OPERATOR_NAMESPACE} already exists"
+
+  # Check if there's an existing ImageStream from the operator deployment
+  if oc get imagestream ${OPERATOR_IMAGE_NAME} -n ${OPERATOR_NAMESPACE} &>/dev/null; then
+    echo "Found existing ImageStream for cli-manager-operator, annotating it..."
+
+    # Get the latest tag
+    LATEST_TAG=$(oc get imagestream ${OPERATOR_IMAGE_NAME} -n ${OPERATOR_NAMESPACE} -o jsonpath='{.spec.tags[0].name}')
+
+    # Annotate the existing ImageStreamTag
+    oc annotate imagestreamtag/${OPERATOR_IMAGE_NAME}:${LATEST_TAG} -n ${OPERATOR_NAMESPACE} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)